src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/CorePersonMigrationService.kt

```diff
@@ -108,16 +113,18 @@ class CorePersonMigrationService(
   }
 }
 
-private fun MigrateCorePersonResponse.toCorePersonMappingsDto(migrationId: String) = CorePersonMappingsDto(
+private fun CreateResponse.toCorePersonMappingsDto(nomisPrisonNumber: String, migrationId: String) = CorePersonMappingsDto(
   mappingType = CorePersonMappingsDto.MappingType.MIGRATED,
   label = migrationId,
-  personMapping = toCorePersonMappingIdDto(),
-  addressMappings = addressIds?.map { it.toCorePersonSimpleMappingIdDto() } ?: emptyList(),
-  phoneMappings = phoneIds?.map { it.toCorePersonPhoneMappingIdDto(CorePersonPhoneMappingIdDto.CprPhoneType.CORE_PERSON) } ?: emptyList(),
-  emailMappings = emailAddressIds?.map { it.toCorePersonSimpleMappingIdDto() } ?: emptyList(),
+  personMapping = CorePersonMappingIdDto(cprId = nomisPrisonNumber, nomisPrisonNumber = nomisPrisonNumber),
+  // TODO check if following lists can send null rather than emptyList
```

Comment:
Lots of TODOs in this PR - as still waiting for core person record team to do updates